EngineeringModel/Views/RuleVerificationListBrowser/RuleVerificationListBrowser.xaml

```diff
@@ -80,7 +80,7 @@
                         </Image>
                     </dx:PixelSnapper>
                     <ContentPresenter x:Name="defaultRowPresenter"
-                                      Content="{Binding Description, Mode=OneWay, UpdateSourceTrigger=PropertyChanged}"
+                                      Content="{Binding}"
```

Comment:
Is this change as expected?

Reply:
The previous code appears to be redundant I therefore replaced it with the code in the above templates. The description is now exposed though a name property which is bound to from the tree grid control.